src/components/Configure/layout/ConditionalProxyLayout/ConditionalProxyLayout.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+
```

Comment:
you're gonna get some messy merge conflicts if you resolve the comments in https://github.com/amp-labs/react/pull/279/files#diff-410e2ed5f3570b278a4c0dbd15eaf8fca546ebc4974107c6a3cb97b9009f7033, since this PR renames the file. 

so to prevent that, feel free to merge this PR into PR 279 first before addressing the comments in that PR. In the future, you might want to avoid renaming a file that a base PR introduces.